### PR TITLE
Example client: Rename `name` to `queueName` to make the usage clear and explicit

### PR DESCRIPTION
**Pull request by andygrunwald**
The client example uses a variable called `name`.
Reading through example, questions will arise "What name is it?".
Especially in the context of AMQP/RabbitMQ this can be the Exchange name, the queue name, the routing key name, the consumer name, etc.

To make the usage clear and explicit, we rename `name` to `queueName`.

### CODE REVIEW

**Review by Zerpet — APPROVED** [review]
Thank you! I'll merge once the merge conflicts are resolved.

**Comment by andygrunwald** [comment]
@Zerpet conflicts fixed.

**Pull request merged**
